src/api/ForecastApi.js

```diff
@@ -0,0 +1,24 @@
+const ForecastApi = {
+
+    fetchAccessToken: async () => {
+        const url = '/auth/get-token';
+
+        return await fetch(url, {
+            method: 'POST',
+            body: JSON.stringify({
+              mode: 'corse'
```

Comment:
CORS is spelled without trailing 'e'. Are you sure you did it right?

Reply:
Yes, sure. I have corrected it.